tests/unit-tests/test_decoder.py

```diff
@@ -981,16 +982,60 @@ def test_MLPEFeatEdgeDecoder(h_dim, feat_dim, out_dim, num_ffn_layers):
         prediction = decoder.predict(g, encoder_feat, efeat)
         pred = out.argmax(dim=1)
         assert_almost_equal(prediction.cpu().numpy(), pred.cpu().numpy())
+
+def check_dropout_train(decoder, in_dim):
+    ones = th.ones(in_dim)
+    decoder.train()
+    output = decoder(ones)
+    assert th.any(output == 0)
+
+def check_dropout_eval(decoder, in_dim):
+    ones = th.ones(in_dim)
+    decoder.eval()
+    output = decoder(ones)
+    assert th.all(output != 0)
+
 @pytest.mark.parametrize("in_dim", [16, 64])
 @pytest.mark.parametrize("out_dim", [1, 8])
 def test_EntityRegression(in_dim, out_dim):
     decoder = EntityRegression(h_dim=in_dim)
     assert decoder.in_dims == in_dim
     assert decoder.out_dims == 1
 
-    decoder = EntityRegression(h_dim=in_dim, out_dim=out_dim)
+    decoder = EntityRegression(h_dim=in_dim, out_dim=out_dim, bias=True)
+    in_tensor = th.ones((1,in_dim))
+    with th.no_grad():
+        decoder.eval()
+        decoder.decoder.data = th.ones((in_dim, out_dim))
+        out = decoder.predict(in_tensor)
```

Comment:
Can you also test the case when decoder.bias is not a zero tensor?
You can hack the decoder.bias for the test.